docker/README.md

```diff
@@ -10,9 +10,11 @@
 
 * docker - Refer to the [official docker installation guide](https://docs.docker.com/install/)
 * git    - Refer to the [official git set-up guide](https://help.github.com/en/github/getting-started-with-github/set-up-git)
-* For base Ubuntu with GPU, install following nvidia container toolkit and driver- 
+* For base Ubuntu with GPU, install following nvidia container toolkit and driver-
   * [Nvidia container toolkit](https://github.com/NVIDIA/nvidia-docker#ubuntu-160418042004-debian-jessiestretchbuster)
   * [Nvidia driver](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/install-nvidia-driver.html)
+  
+* NOTE - Dockerfiles have not been tested on windows native platform.
```

Comment:
I don't understand the point of this. Is there something stopping us from testing on Windows native? Or is it that these have been tested on WSL but not native?

Reply:
We don't have win10 etc. and testing docker on EC2 win server 2019 is not supported. Please refer my comment in PR description hence this comment.